### PR TITLE
fix(deps): upgrade core version to address jsonwebtoken vulnerability

### PR DESCRIPTION
**Pull request by apaparazzi0329**

### CODE REVIEW

File: test/integration/discovery.v1.test.js [hunk 1]
```diff
@@ -93,7 +93,6 @@ describe('discovery v1 integration', () => {
   it('should listCollections()', async () => {
     const params = {
       environmentId,
-      configurationId: configurationId,
```
Comment (jeff-arn) [inline test/integration/discovery.v1.test.js:96]: why is `configurationId` removed?

File: test/integration/discovery.v1.test.js [hunk 2]
```diff
@@ -105,12 +104,10 @@ describe('discovery v1 integration', () => {
   it('should perform a federated query for notices', async () => {
     const params = {
       environmentId,
-      configurationId: configurationId,
```
Comment (jeff-arn) [inline test/integration/discovery.v1.test.js:108]: same as above

File: test/integration/discovery.v1.test.js [hunk 3]
```diff
@@ -105,12 +104,10 @@ describe('discovery v1 integration', () => {
...
       collectionIds: [collectionId, collectionId2],
       filter: 'yesplease',
       count: 10,
       sort: ['+field_1', '-field_2'],
-      natural_language_query: 'a question about stuff and things',
```
Comment (jeff-arn) [inline test/integration/discovery.v1.test.js:113]: why is `natural_language_query` removed?

File: test/integration/discovery.v1.test.js [hunk 4]
```diff
@@ -405,7 +402,6 @@ describe('discovery v1 integration', () => {
       expect(Array.isArray(result.aggregations)).toBe(true);
       expect(result.aggregations[0].results).toBeDefined();
       expect(Array.isArray(result.aggregations[0].results)).toBe(true);
-      expect(result.aggregations[0].results[0].matching_results).toBeDefined();
```
Comment (jeff-arn) [inline test/integration/discovery.v1.test.js:408]: why is `matching_results` removed?

File: test/integration/discovery.v1.test.js [hunk 5]
```diff
@@ -418,7 +414,6 @@ describe('discovery v1 integration', () => {
       expect(Array.isArray(result.aggregations)).toBe(true);
       expect(result.aggregations[0].results).toBeDefined();
       expect(Array.isArray(result.aggregations[0].results)).toBe(true);
-      expect(result.aggregations[0].results[0].event_rate).toBeDefined();
```
Comment (jeff-arn) [inline test/integration/discovery.v1.test.js:421]: why is `event_rate` removed?

File: test/integration/text-to-speech.test.js [hunk 6]
```diff
@@ -53,7 +54,14 @@ describe('text to speech_integration', () => {
       });
 
       synthStream.on('close', (code, reason) => {
-        done();
+        console.log(code);
```
Comment (jeff-arn) [inline test/integration/text-to-speech.test.js:57]: remove `console.log

File: test/integration/text-to-speech.test.js [hunk 7]
```diff
@@ -53,7 +54,14 @@ describe('text to speech_integration', () => {
...
+        if (code === 1000) {
+          done();
+        } else if (code === undefined) {
+          // Do nothing: this is in order to prevent done() being called twice
```
Comment (jeff-arn) [inline test/integration/text-to-speech.test.js:61]: I see the comment inline, but can you elaborate more on this? It seems that normally if `code === undefined` we would go to the `else` block and call `fail();`. Do we not want to fail the test if it is `undefined`?

**Review by jeff-arn — COMMENTED** [review]
I left some questions inline on the PR

**Comment by jeff-arn** [comment]
@apaparazzi0329 also can you update the Readme in this PR to correct what the minimum version of node is?

File: test/integration/discovery.v1.test.js (hunk 1 above)
Comment (apaparazzi0329) [reply test/integration/discovery.v1.test.js:96]: This is no longer a valid parameter

File: test/integration/discovery.v1.test.js (hunk 3 above)
Comment (apaparazzi0329) [reply test/integration/discovery.v1.test.js:113]: This is no longer a valid parameter

File: test/integration/text-to-speech.test.js (hunk 6 above)
Comment (apaparazzi0329) [reply test/integration/text-to-speech.test.js:57]: Whoops good catch

File: test/integration/text-to-speech.test.js (hunk 7 above)
Comment (apaparazzi0329) [reply test/integration/text-to-speech.test.js:61]: Normally yes but this undefined is actually a duplicate call which creates a duplicate `done` call which is no longer valid

File: test/integration/discovery.v1.test.js (hunk 5 above)
Comment (apaparazzi0329) [reply test/integration/discovery.v1.test.js:421]: event_rate is an optional property and is null in this case

File: test/integration/text-to-speech.test.js (hunk 7 above)
Comment (jeff-arn) [reply test/integration/text-to-speech.test.js:61]: Why is there a duplicate call on the `close` event?

Comment (apaparazzi0329) [reply test/integration/text-to-speech.test.js:60]: Your guess is as good as mine but I did test the current version and this has already existed and only came up now due to updating Jest

Comment (jeff-arn) [reply test/integration/text-to-speech.test.js:60]: Hmm ok

**Review by jeff-arn — APPROVED** [review]
LGTM

**Pull request merged**